src/internal/connector/onedrive/permission.go

```diff
@@ -207,12 +209,12 @@ func UpdatePermissions(
 
 		pbody.SetRecipients([]models.DriveRecipientable{rec})
 
-		np, err := service.Client().DrivesById(driveID).ItemsById(itemID).Invite().Post(ctx, pbody, nil)
+		newPerm, err := api.PostItemPermissionUpdate(ctx, service, driveID, itemID, pbody)
 		if err != nil {
-			return graph.Wrap(ctx, err, "setting permissions")
+			return err
```

Comment:
will we get nice stack trace info that shows this is where `PostItemPermissionUpdate` was called if we don't wrap the error here? I haven't really looked through all of the stack trace generation code to know

Reply:
Stack trace generation is attached to any clues wrap or stack call.  So we can stack here, too, if wanted.